code-challenges.test.js

```diff
@@ -34,26 +49,122 @@ const object3 = { number: -7 }
+// Give the funciton the parameter object for clarity.
+// Use the arrow funciton syntex.
+// Create a condtional using an if statement to test if the obeject is divisible by three, which will be set to strictly equal to 0.
+// Create an else statement as a catch all for all other numbers that do not meet the criteria above.
+// Set both the if and else statements to return a string interpolated message ' object.number is divisible by three, or object.numer is not divisible by three.
+// DO NOT include a console.log because it is being tested!!!
+
+const divBYThree = (object) => {
+    if(object.number % 3 === 0){
+      return `${object.number} is divisible by three.`
+    }else{
+        return `${object.number} is not divisible by three.`
+    }
+  }
+
+//Output:
+//   PASS  ./code-challenges.test.js
+//   PASS  ./jest.test.js
+
+//  Test Suites: 2 passed, 2 total
+//  Tests:       2 passed, 2 total
 
-// --------------------3) Create a function that takes in a string and logs the index of the first vowel.
+// --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const vowelTester1 = "learn"
-// Expected output: 1
+// Psuedo:
+// Take the test from question one as a template.
+// Change the name of the function to allCap.
+// Change 'it' to what the function will do.
+// Create two expect statements for both of the arrays given to be arugments in the function.
+
+describe("allCap", () => {
+  it("function will take in the given arrays as arguments and return new arrays with the strings in all capital", () => {
+    expect(allCap(["streetlamp", "potato", "teeth", "conclusion", "nephew"])).toEqual(["STREETLAMP", "POTATO", "TEETH", "CONCLUSION", "NEPHEW"])
+    expect(allCap(["temperature", "database", "chopsticks", "mango"])).toEqual(["TEMPERATURE", "DATABASE", "CHOPSTICKS", "MANGO"])
+  })
+})
+
+// const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+// // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+// const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+// // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+// // b) Create the function that makes the test pass.
+
+// // Psuedo:
+// // Create a function named allCap that was named in the test.
+// // Give the function a parameters called array to match the data type.
+// // Add the arrow syntex for the funciton.
+// // Use method .map() to iterate through each array
+// // .map() will create a new array. Add the .toUpperCase method to change the strings stored in each array to all upper case letters.
+//The return will have the toUpperCase method to avoid using another  line of code.
+// 
+
+const allCap = (array) => {
+return array.map((value) => {
+  return value.toUpperCase()
```

Comment:
Awesome, you got your desire output. I encourage you to refactor and consider the given output. How can you make only the first letter of each work capitalized?